api_v3/lib/types/filters/ValidateAccessResponseProfile.php

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ * @package api
+ * @subpackage filters
+ */
+
+class ValidateAccessResponseProfile
+{
+
+	const LIST_ACTION = 'listAction';
+	const COMMENTS = '#@(.*?)\n#s';
+	const RELATED_SERVICE = 'relatedService';
+
+	/**
+	 * @param KalturaRelatedFilter $relatedFilter
+	 * @return bool
+	 * @throws Exception
+	 */
+	public static function validateAccess($relatedFilter)
+	{
+		$clazz = self::getServiceClassInstance($relatedFilter);
+		return self::validate($clazz);
+	}
+
+	/**
+	 * @param $relatedFilter
+	 * @return string $clazz
+	 * @throws Exception
+	 */
+	private static function getServiceClassInstance($relatedFilter)
+	{
+		$clazz = self::getServiceClazz($relatedFilter);
+		/** if Class is not found then do not allow the response profile filter to get the response  */
+		if (!class_exists($clazz) || !method_exists($clazz, self::LIST_ACTION)) {
```

Comment:
Either remove this test or change error message to state that list action is missing,